provider_packages/SETUP_TEMPLATE.py.jinja2

```diff
@@ -0,0 +1,102 @@
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE\
+# OVERWRITTEN WHEN RUNNING
+#
+# ./breeze prepare-provider-readme
+#
+# IF YOU WANT TO MODIFY IT, YOU SHOULD MODIFY THE TEMPLATE
+# `SETUP_TEMPLATE.py.jinja2` IN the `provider_packages` DIRECTORY
+
+"""Setup.py for the {{ PACKAGE_PIP_NAME }} package."""
+
+import io
+import logging
+import os
+import sys
+
+from os.path import dirname
+from setuptools import find_packages, setup
+
+logger = logging.getLogger(__name__)
+
+version = '{{ RELEASE_NO_LEADING_ZEROS }}{{VERSION_SUFFIX }}'
+
+my_dir = dirname(__file__)
+
+try:
+    with io.open(os.path.join(my_dir,
+                              '{{ PROVIDER_PATH }}/{{ README_FILE }}'),
+                 encoding='utf-8') as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = ''
+
+
+def do_setup(version_suffix_for_pypi=''):
+    """
+    Perform the package setup.
+    """
+    setup(
+        name='{{ PACKAGE_PIP_NAME }}',
+        description='{{ PROVIDER_TYPE }} package {{ PACKAGE_PIP_NAME }} for Apache Airflow',
+        long_description=long_description,
+        long_description_content_type='text/markdown',
+        license='Apache License 2.0',
+        version=version + version_suffix_for_pypi,
+        packages=find_packages(
+            include=['airflow.providers.{{ PROVIDER_PACKAGE_ID }}*']),
+        zip_safe=False,
+        install_requires={{ INSTALL_REQUIREMENTS }},
+        setup_requires={{ SETUP_REQUIREMENTS }},
+        extras_require={{ EXTRAS_REQUIREMENTS }},
+        classifiers=[
+            'Development Status :: 5 - Production/Stable',
+            'Environment :: Console',
+            'Environment :: Web Environment',
+            'Intended Audience :: Developers',
+            'Intended Audience :: System Administrators',
+            'License :: OSI Approved :: Apache Software License',
+            'Programming Language :: Python :: 3.6',
+            'Programming Language :: Python :: 3.7',
+            'Programming Language :: Python :: 3.8',
+            'Topic :: System :: Monitoring',
+        ],
+        author='Apache Software Foundation',
+        author_email='dev@airflow.apache.org',
+        url='http://airflow.apache.org/',
+        download_url=(
+            'https://dist.apache.org/repos/dist/release/airflow/{{ PROVIDERS_FOLDER }}' + version),
+        python_requires='~=3.6',
+    )
+
+#
+# Note that --version-suffix-for-pypi should only be used in case we generate RC packages for PyPI
+# Those packages should have actual RC version in order to be published even if source version
+# should be the final one.
+#
+if __name__ == "__main__":
+    suffix=''
+    if len(sys.argv) > 0 and sys.argv[1] == "--version-suffix-for-pypi":
```

Comment:
Argh, sorry, you meant
```suggestion
    if len(sys.argv) > 1 and sys.argv[1] == "--version-suffix-for-pypi":
```

```
Traceback (most recent call last):
  File "setup.py", line 101, in <module>
    if len(sys.argv) > 0 and sys.argv[1] == "--version-suffix-for-pypi":
IndexError: list index out of range
```

Reply:
Ah yeah I did test it and corrected it . But in the generated setup.py not the template :(

Reply:
My bad, sorry. I reset, regenerated from scratch and it looks good now:

```
[jarek:~/code/airflow/airflow/providers/google] fix-setup-py-in-provider-packages(+42/-4)+* ± ls
ads					  BACKPORT_PROVIDERS_CHANGES_2020.10.05.md  cloud   config_templates  __init__.py	  PROVIDERS_CHANGES_0.0.1.md  setup.cfg  suite
BACKPORT_PROVIDERS_CHANGES_2020.06.24.md  BACKPORT_README.md			    common  firebase	      marketing_platform  README.md		      setup.py
[jarek:~/code/airflow/airflow/providers/google] fix-setup-py-in-provider-packages(+42/-4)+* ± python setup.py 
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied

```

Reply:
Perfect, thanks